ptz_relative: true
cmd_topic_ptz: /guardian/command_ptz
tilt_increment_degrees: 3.0        # in degrees (for 100ms scans -> tilt_range_degrees / (period * 10))
epsilon: 0.05
period: 4.0 # seconds for moving from lower limit to upper limit


# looking at floor
#laser_upper_limit_degrees: 40       # floor (50) | 20
#laser_lower_limit_degrees: 10      # ceiling (-80) | 15

# looking at ceiling
laser_upper_limit_degrees: 45
laser_lower_limit_degrees: -75
